airdrop/processor/loyalty_airdrop.py

```diff
@@ -8,6 +9,11 @@ def __init__(self, airdrop_id, airdrop_window_id=None):
         self.airdrop_window_id = airdrop_window_id
         self.register_all_window_at_once = True
         self.domain_name = "SingularityNet"
+        self.chain_context = {
+            "deposit_address": LoyaltyAirdropConfig.deposit_address,
+            "amount": LoyaltyAirdropConfig.pre_claim_transfer_amount.value["amount"],
```

Comment:
Thinking if we wanna move this to a static table in DB , so if there is a need to change the amount / deposit address we don’t need a deployment agree ?

Reply:
We have made a decision in the starting that one airdrop will be associated with one blockchain environment. So thought of keeping it at airdrop level.